src/__tests__/to-contain-element.tsx

```diff
@@ -16,6 +16,7 @@ const parent = queryByTestId('parent');
 const child = queryByTestId('child');
 const textElement = queryByTestId('text-element');
 const nonExistantElement = queryByTestId('not-exists');
+const fakeElement = null;
```

Comment:
```suggestion
const fakeElement = undefined;
```

Reply:
It was due to a wrong type on my side, fixed it with the rebase